model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -117,6 +117,25 @@ class LicenseInfoResolver(
                         )
                     )
                 }
+
+                licenseInfo.declaredLicenseInfo.copyrightHolders.takeIf { it.isNotEmpty() }?.let {
+                    locations.add(
+                        ResolvedLicenseLocation(
+                            provenance = UnknownProvenance,
+                            location = UNDEFINED_TEXT_LOCATION,
+                            appliedCuration = null,
+                            matchingPathExcludes = emptyList(),
+                            copyrights = licenseInfo.declaredLicenseInfo
+                                .copyrightHolders.mapTo(mutableSetOf()) { copyrightHolder ->
+                                    ResolvedCopyrightFinding(
+                                        statement = copyrightHolder,
```

Comment:
Related to my question above: As you're not adding "Copyright" here (like done for `authors`), I assume you expect `copyrightHolder` to already contain that prefix, correct?